### ISSUE TITLE
Bpod light  still on following Firmware update

### ISSUE THREAD

**Issue by GaelleChapuis** [int-brain-lab/iblrig#80]
See slack message in troubleshooting channel.

**Comment by nbonacchi** [comment]
will be fixed in next release

**Comment by HernandoMV** [comment]
Version is current:

```
Current version: 3.5.3

Available versions: ['2.0.0', '2.0.1', '3.0.0', '3.1.0', '3.1.1', '3.2.0', '3.2.1', '3.2.2', '3.2.3', '3.2.4', '3.3.0', '3.4.0', '3.4.1', '3.5.0', '3.5.1', '3.5.2', '3.5.3']


The version you have checked out is the latest version
```


But Bpod light is still on. What do to?

**Comment by HernandoMV** [comment]
They are off when the protocol start. All good
